free-programming-books.md

```diff
@@ -1665,7 +1665,11 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
   * [CakePHP Cookbook 2.x](http://book.cakephp.org/2.0/_downloads/en/CakePHPCookbook.pdf) (PDF)
 * Drupal Framework
   * Drupal 7
+    * [Master Drupal 7 in 7 hours](https://dl.dropboxusercontent.com/u/54624702/Master%20Drupal%20in%207%20hours_v1.1.pdf) (PDF)
```

Comment:
Please replace this link, the [guidelines](https://github.com/vhf/free-programming-books/blob/master/CONTRIBUTING.md#guidelines) reject any file hosting service.